opus/application/apps/help/templates/help/gettingstarted.html

```diff
@@ -220,13 +220,15 @@ <h3>Browsing</h3>
     can be changed by clicking on <b>Select Metadata</b> at the top. Fields may
     be added or removed by clicking on them on the left or removed by clicking
     on the <i class="far fa-trash-alt"></i> on the right, and may be reordered
-    by dragging. Fields can be added or removed in the table view by
-    hovering over the column header; click on the <i class="far fa-trash-alt"></i>
+    by dragging. If available, the units a metadata field is displayed in can be
```

Comment:
I think this may be better worded like this:

When available, the display units for the metadata field can be changed by clicking on the current units name shown in the parentheses.

Reply:
I think I would get rid of "the" and say: When available, the display units for the metadata field can be changed by clicking on the current units name shown in parentheses.